Jint/Runtime/Debugger/DebugCallStack.cs

```diff
@@ -0,0 +1,45 @@
+﻿using System;
+using System.Collections;
+using System.Collections.Generic;
+using System.Linq;
+using Esprima;
+using Esprima.Ast;
+using Jint.Native;
+using Jint.Runtime.CallStack;
+using Jint.Runtime.Environments;
+
+namespace Jint.Runtime.Debugger
+{
+    public class DebugCallStack : IReadOnlyList<CallFrame>
+    {
+        private List<CallFrame> _stack;
+        public CallFrame this[int index] => _stack[index];
+
+        public int Count => _stack.Count;
+
+        internal DebugCallStack(Engine engine, Location location, JintCallStack callStack, JsValue returnValue)
+        {
+            _stack = new List<CallFrame>();
```

Comment:
could initialize this to size of `callStack.Stack.Count + 1`

Reply:
Yeah - went for `callStack.Count` now (`callStack.Stack` is `IEnumerable`).